### PR TITLE
docs: Add multi owner support for GitHub OIDC Trust

### PR DESCRIPTION
**Pull request by mariux**

### CODE REVIEW

**Review by sixstone-qq — APPROVED** [review]
LGTM

**Review by soerenmartius — COMMENTED** [review]
@AnnuCode could you proof read this pls?

File: docs/cloud/organization/settings.md [hunk 1]
```diff
@@ -7,11 +7,18 @@
...
-    - a GitHub Repository to trust a single repository within a GitHub Organization of the form `owner/repository`
-- **Slack WebHook URL** - This field is written only and will not display the currently set value. It can be used to set a new WebHook URL. To generate a Slack WebHook URL, please follow the instructions in the [Slack WebHook Documentation](https://opentffoundation.slack.com/apps/A0F7XDUAZ-incoming-webhooks). *This integration will be replaced with a Slack App.*
+
+  - a GitHub Organization/Owner to trust all repositories within a GitHub Organization of the form `owner`
+  - a comma-separated list of GitHub Organizations/Owners of the form `ownerA,ownerB,ownerC`
+  - a GitHub Repository to trust a single repository within a GitHub Organization of the form `owner/repository`
+
+- **Slack WebHook URL** - This field is written only and will not display the currently set value. It can be used to set a new WebHook URL.
+  To generate a Slack WebHook URL, please follow the instructions in the [Slack WebHook Documentation](https://opentffoundation.slack.com/apps/A0F7XDUAZ-incoming-webhooks).
```
Comment (AnnuCode) [inline docs/cloud/organization/settings.md:20]: just a small thought: the word 'please' can be avoided here.

**Review by soerenmartius — APPROVED** [review]
lgtm

**Pull request merged**